contrib/collectd/files/collectd

```diff
@@ -0,0 +1,7 @@
+# collectd service
+
+type        = process
+command     = /usr/bin/collectd -f
+pid-file    = /var/run/collectd.pid
```

Comment:
```suggestion
```

this doesn't do anything for type = process

Reply:
Hmm, I tried bgprocess, and it hung on `dinitctl stop` even though I see in their code that it's responsive to sigterm.  I'll look into this a bit more, your suggestion is probably the fix.

Reply:
i'm just not sure why you added that in the first place, since with -f it never creates this file, and dinit ignores the statement entirely for =process